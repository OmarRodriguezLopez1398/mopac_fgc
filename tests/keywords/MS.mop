UHF MS=1.5
comment1
comment2
N  0.0 0.0 0.0

RHF MS=1.5 C.I.=(3,1) MECI
comment1
comment2
N 0.0 0.0 0.0

RHF MS=1.5 OPEN(3,3) MECI
comment1
comment2
N 0.0 0.0 0.0
