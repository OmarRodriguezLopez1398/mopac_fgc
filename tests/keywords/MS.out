 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 21.298M, James J. P. Stewart,   **
 **                           web-site: HTTP://OpenMOPAC.net.                 **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Fri Oct 29 13:45:34 2021  *
 *  UHF        - UNRESTRICTED HARTREE-FOCK CALCULATION
 *  MS=        - IN MECI, MAGNETIC COMPONENT OF SPIN =  1.5
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *******************************************************************************
UHF MS=1.5
 comment1
 comment2
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       N          0.00000000  *   0.00000000  *   0.00000000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         N          0.0000      0.0000      0.0000

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: N  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


          UHF CALCULATION, NO. OF ALPHA ELECTRONS =    4
                           NO. OF BETA  ELECTRONS =    1

          Geometry optimization using EF

     GRADIENT =  0.00000 IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
 UHF MS=1.5
 comment1
 comment2


     GRADIENTS WERE INITIALLY ACCEPTABLY SMALL                
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 21.298M)
                                                       Fri Oct 29 13:45:34 2021




          FINAL HEAT OF FORMATION =        113.00000 KCAL/MOL =     472.79200 KJ/MOL


          COSMO AREA              =         42.08 SQUARE ANGSTROMS
          COSMO VOLUME            =         25.67 CUBIC ANGSTROMS

          IONIZATION POTENTIAL    =         14.093314 EV
          ALPHA SOMO LUMO (EV)    =        -14.093
          BETA  SOMO LUMO (EV)    =        -21.026  0.027
          NO. OF ALPHA ELECTRONS  =          4
          NO. OF BETA  ELECTRONS  =          1
          MOLECULAR WEIGHT        =         14.0067         POINT GROUP:  R3  


          SCF CALCULATIONS        =          1
 
          WALL-CLOCK TIME         =          0.008 SECONDS
          COMPUTATION TIME        =          0.009 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       N          0.00000000  *   0.00000000  *   0.00000000  *

                             CARTESIAN COORDINATES

   1    N        0.000000000     0.000000000     0.000000000


           Empirical Formula: N  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


          ALPHA EIGENVALUES  
 -29.95821 -14.09331 -14.09331 -14.09331


           BETA EIGENVALUES 
 -21.02591   0.02702   0.02702   0.02702


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          N           0.000000        5.0000     2.00000     3.00000
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000     0.000     0.000     0.000
 HYBRID        -0.000     0.000     0.000     0.000
 SUM           -0.000     0.000     0.000     0.000


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  N   2.00000   1.00000   1.00000   1.00000


                    (SZ)    =    1.500000
                    (S**2)  =    3.750000


          ATOMIC ORBITAL SPIN POPULATIONS

    Atom   Total    s     px     py     pz   
         Atomic
    1  N  3.0000  0.000  1.000  1.000  1.000
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 21.298M, James J. P. Stewart,   **
 **                           web-site: HTTP://OpenMOPAC.net.                 **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Fri Oct 29 13:45:34 2021  *
 *  RHF        - RESTRICTED HARTREE-FOCK CALCULATION
 *  C.I.=(N,M) - 1 DOUBLY FILLED LEVELS USED IN A C.I. INVOLVING 3 M.O.'S
 *  MS=        - IN MECI, MAGNETIC COMPONENT OF SPIN =  1.5
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *  MECI       - M.E.C.I. WORKING TO BE PRINTED
 *******************************************************************************
RHF MS=1.5 C.I.=(3,1) MECI
 comment1
 comment2
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       N          0.00000000  *   0.00000000  *   0.00000000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         N          0.0000      0.0000      0.0000

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: N  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  

      THERE ARE    2 DOUBLY FILLED LEVELS


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 2


                       NO. OF SINGLY OCCUPIED LEVELS =    1

          Geometry optimization using EF

     GRADIENT =  0.00000 IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
 RHF MS=1.5 C.I.=(3,1) MECI
 comment1
 comment2


     GRADIENTS WERE INITIALLY ACCEPTABLY SMALL                
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 21.298M)
                                                       Fri Oct 29 13:45:34 2021




          FINAL HEAT OF FORMATION =        113.00000 KCAL/MOL =     472.79200 KJ/MOL


          COSMO AREA              =         42.08 SQUARE ANGSTROMS
          COSMO VOLUME            =         25.67 CUBIC ANGSTROMS

          IONIZATION POTENTIAL    =         13.138217 EV          STATE:   1 QUARTET S(u)
          HOMO (SOMO) LUMO (EV)   =      -10.750 ( -7.033) -3.316
          NO. OF FILLED LEVELS    =          2
          AND NO. OF OPEN LEVELS  =          1
          MOLECULAR WEIGHT        =         14.0067         POINT GROUP:  R3  


          SCF CALCULATIONS        =          1
 
          WALL-CLOCK TIME         =          0.016 SECONDS
          COMPUTATION TIME        =          0.017 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       N          0.00000000  *   0.00000000  *   0.00000000  *

                             CARTESIAN COORDINATES

   1    N        0.000000000     0.000000000     0.000000000


           Empirical Formula: N  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


                  EIGENVALUES  
 -25.49206 -10.75048  -7.03315  -3.31582


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          N           0.000000        5.0000     2.00000     3.00000
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000     0.000     0.000     0.000
 HYBRID         0.000     0.000     0.000     0.000
 SUM            0.000     0.000     0.000     0.000


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  N   2.00000   1.00000   1.00000   1.00000


          MULTI-ELECTRON CONFIGURATION INTERACTION CALCULATION


          COMPONENT OF SPIN  =  1.5


           NO OF CONFIGURATIONS CONSIDERED = 1

 CONFIGURATIONS CONSIDERED IN C.I.      
 M.O. NUMBER :         2   3   4
          ENERGY

              1        1   1   1
         -6.8729       0   0   0

  STATE       ENERGY (EV)        Q.N.  SPIN   SYMMETRY              POLARIZATION
         ABSOLUTE     RELATIVE                                X           Y           Z

    1+   -6.872921    0.000000     1+  QUARTET     S(u)

           'RELATIVE ENERGY' Is relative to the lowest state calculated.
           This may or may not be the ground state.
           The "+" symbol indicates the root used.
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 21.298M, James J. P. Stewart,   **
 **                           web-site: HTTP://OpenMOPAC.net.                 **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Fri Oct 29 13:45:34 2021  *
 *  RHF        - RESTRICTED HARTREE-FOCK CALCULATION
 *  OPEN(M,N)  - THERE ARE 3 ELECTRONS IN 3 LEVELS
 *  MS=        - IN MECI, MAGNETIC COMPONENT OF SPIN =  1.5
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *  MECI       - M.E.C.I. WORKING TO BE PRINTED
 *******************************************************************************
RHF MS=1.5 OPEN(3,3) MECI
 comment1
 comment2
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       N          0.00000000  *   0.00000000  *   0.00000000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         N          0.0000      0.0000      0.0000

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: N  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  

      THERE ARE    1 DOUBLY FILLED LEVELS


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 1


                       NO. OF SINGLY OCCUPIED LEVELS =    3

          Geometry optimization using EF

     GRADIENT =  0.00000 IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
 RHF MS=1.5 OPEN(3,3) MECI
 comment1
 comment2


     GRADIENTS WERE INITIALLY ACCEPTABLY SMALL                
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 21.298M)
                                                       Fri Oct 29 13:45:34 2021




          FINAL HEAT OF FORMATION =        113.00000 KCAL/MOL =     472.79200 KJ/MOL


          COSMO AREA              =         42.08 SQUARE ANGSTROMS
          COSMO VOLUME            =         25.67 CUBIC ANGSTROMS

          GRADIENT NORM           =          0.00000          =       0.00000 PER ATOM
          NO. OF FILLED LEVELS    =          1
          AND NO. OF OPEN LEVELS  =          3
          MOLECULAR WEIGHT        =         14.0067         POINT GROUP:  R3  


          SCF CALCULATIONS        =          5
 
          WALL-CLOCK TIME         =          0.027 SECONDS
          COMPUTATION TIME        =          0.028 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       N          0.00000000  *   0.00000000  *   0.00000000  *

                             CARTESIAN COORDINATES

   1    N        0.000000000     0.000000000     0.000000000


           Empirical Formula: N  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


                  EIGENVALUES  
 -25.49206  -7.03315  -7.03315  -7.03315


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          N           0.000000        5.0000     2.00000     3.00000
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000     0.000     0.000     0.000
 HYBRID         0.000     0.000     0.000     0.000
 SUM            0.000     0.000     0.000     0.000


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  N   2.00000   1.00000   1.00000   1.00000


          MULTI-ELECTRON CONFIGURATION INTERACTION CALCULATION


          COMPONENT OF SPIN  =  1.5


           NO OF CONFIGURATIONS CONSIDERED = 1

 CONFIGURATIONS CONSIDERED IN C.I.      
 M.O. NUMBER :         2   3   4
          ENERGY

              1        1   1   1
        -10.5902       0   0   0

  STATE       ENERGY (EV)        Q.N.  SPIN   SYMMETRY              POLARIZATION
         ABSOLUTE     RELATIVE                                X           Y           Z

    1+  -10.590250    0.000000     1+  QUARTET     S(u)

           'RELATIVE ENERGY' Is relative to the lowest state calculated.
           This may or may not be the ground state.
           The "+" symbol indicates the root used.

 **********************
 *                    *
 * JOB ENDED NORMALLY *
 *                    *
 **********************



 TOTAL JOB TIME:             0.06 SECONDS

 == MOPAC DONE ==
